src/ui/page_wifi.c

```diff
@@ -316,6 +316,10 @@ static void page_wifi_update_settings() {
             system_script(WIFI_STA_ON);
         }
     }
+
+    if (complete_callback != NULL) {
+        complete_callback();
```

Comment:
Same issue here as well.... what happens if this is invoked by the main thread and we keep executing nested complete_callback functions.... this could potentially cause the main thread to die and could make adding new deferred functions difficult because we will not know why initially valid new functionality is crashing because the stack memory or frames have been exhausted.